OSDOCS-4587: Apply Quick filters to network traffic data (#53954)
openshift/openshift-docs: modules/network-observability-quickfilter.adoc

= Filtering network traffic
By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the FlowCollector instance. You can use the filter options to observe the required data by changing the preset filter. Following are the available filter options:

.Quick filters

> The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can select or select the options from console. 

Reviewer: A word must be missing in the last sentence?

.Advanced filters
To set advanced filters:

* Select the parameters. The default selected parameter is *Common Namespaces*.
* Enter the value. To filter the data that matches the text value exactly, enter the value in double quotation marks. 

> The section *Common* in the parameter drop-down applies filter to both source and destination. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.

Reviewer: Instead of
> applies filter to both source and destination.

I would say:
> applies filter to either source or destination.

(For example, "common namespace = foo" will match flows where source namespace=foo or dest namespace = foo, but it hasn't to be both at the same time)

.Query Options
You can use *Query Options* to optimize the search results, as listed below:

> ** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node.You can select either *Source* or *Destination*. The option *Both* is disabled for topology view. The default selected value is *Destination*.

Reviewer: Mentioning also overview view where both is also disabled:

```suggestion
** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for overview and topology views. The default selected value is *Destination*.
```

> ** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node.You can select either *Source* or *Destination*. The option *Both* is disabled for overview and topology view. The default selected value is *Destination*.

Reviewer 2: Missing space after `.`

```suggestion
** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for overview and topology view. The default selected value is *Destination*.
```

> ** *Match filters*: You can determine the relation between different filter parameters selected in advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered.The default value is *Match all*.

Reviewer 2: ```suggestion
** *Match filters*: You can determine the relation between different filter parameters selected in advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
```
